src/scripts/scriptlib.fsx

```diff
@@ -172,7 +172,7 @@ module Scripting =
 
     let redirectToLog () = redirectTo System.Console.Out
 
-#if !FSHARP_SUITE_DRIVES_CORECLR_TESTS
+#if !NETCOREAPP
     let defaultPlatform = 
         match Environment.OSVersion.Platform, Environment.Is64BitOperatingSystem with 
         | PlatformID.MacOSX, true -> "osx.10.11-x64"
```

Comment:
I think both of these string values are suspect since they refer to specific, quite old versions of their respective OS's. Should this be updated?

Reply:
Is the value referenced anywhere?

Reply:
@cartermp, the older platform designation is fine, since future platforms are backwards compatible.

It's an autoopen module so, figuring out where the value is used, is tougher.  But if someone took the trouble to look, it would be usefull.